easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1830,6 +1830,30 @@ def get_cuda_cc_template_value(self, key):
             raise EasyBuildError(error_msg, key)
 
 
+def get_parallel_ec_param_value(cfg, log):
+    """
+    Get value for 'parallel' easyconfig parameter for given EasyConfig instance.
+    Takes into account:
+    * --parallel EasyBuild configuration option (if defined)
+    * 'parallel' easyconfig parameter (if defined)
+    * 'maxparallel' easyconfig parameter (if defined)
+    * number of available cores (incl. affinity of active EasyBuild session)
+    """
+    # set level of parallelism for build
+    par = build_option('parallel')
+    cfg_par = cfg['parallel']
+    if cfg_par is None:
+        log.debug("Desired parallelism specified via 'parallel' build option: %s", par)
```

Comment:
Can't par be None here?
If there is no build_option('parallel') and cfg_par is None?

If it can't you wouldn't check against None on line 1847...

Reply:
`par = None` is handled by  `det_parallelism`, so that is fine and means "auto-detect from #cpus"

Reply:
My objection was that it would print "Desired parallelism specified via 'parallel' build option: None" here if par is None and cfg_par is None

Reply:
Yeah, you're right, didn't think of it that way.